pkg/handlers/config.go

```diff
@@ -635,11 +648,11 @@ func (h *Handler) CurrentAppConfig(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if renderedConfig != nil {
-		configGroups = renderedConfig.Spec.Groups
+		currentAppConfigResponse.ConfigGroups = renderedConfig.Spec.Groups
+		currentAppConfigResponse.ValidationErrors = kotsadmvalidator.ValidateConfigSpec(renderedConfig.Spec)
 	}
 
 	currentAppConfigResponse.Success = true
-	currentAppConfigResponse.ConfigGroups = configGroups
```

Comment:
this means that the config groups will be null instead of an empty array when returned to the UI which might break

Reply:
setting `currentAppConfigResponse.ConfigGroups = []kotsv1beta1.ConfigGroup{}` before so it works as it is